3rdparty/udis86/libudis86/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# UJIT: CMake file for udis86 to avoid dependencies on autoconf and libtool.
+# UJIT: Based on https://github.com/vmt/udis86/pull/99/files by hasherezade.
+# UJIT: Copyright (C) 2020 Anton Soldatov.
+
+set(ITAB_H "${CMAKE_CURRENT_LIST_DIR}/itab.h")
+set(ITAB_C "${CMAKE_CURRENT_LIST_DIR}/itab.c")
+
+if(NOT EXISTS ${ITAB_H} OR NOT EXISTS ${ITAB_C})
+  message(FATAL_ERROR "You must generate files: ${ITAB_H}, ${ITAB_C} before you start.")
```

Comment:
How are itab.h/itab.c generated? If by some script (I presume), then it should be some CMake's custom command and the target which uses these files should depend on it. The user shouldn't "generate" this files, they should be generated automatically by CMake and if it fails to find them, it should just report that they're missing.

Reply:
Generation of `itab.{c,h}` was already hardcoded some time ago to avoid build-time dependencies on Python 2. I will simply remove this confusing warning, it should never fire unless our build system is somehow corrupted.